### PR TITLE
Submitting lesson02 Assignment "Grid Printer Exercise"

### PR DESCRIPTION
**Pull request by ldenney89**
This assignment is in the lesson02 directory, named "grid_printer.py". 

It can be run as a main file. 

Submitting assignment.

Edit: included is fizzbuzz.py, series.py

### CODE REVIEW

**Comment by ldenney89** [comment]
Also, included is fizzbuzz.py, the second part of lesson02 assignment.

**Comment by ldenney89** [comment]
added series.py, the fibonacci part of lesson02

File: students/laura_denney/lesson02/fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#-------------------------------------------------#
+# Title: Fizz Buzz
+# Dev:   LDenney
+# Date:  October 3, 2018
+# ChangeLog: (Who, When, What)
+#   Laura Denney, 10/3/18, Created File
+#-------------------------------------------------#
+
+def fizzbuzz():
```
Comment (charlesrobison) [inline students/laura_denney/lesson02/fizzbuzz.py]: You'll want to use snake_case for function names per PEP8: fizz_buzz():

File: students/laura_denney/lesson02/fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            print("FizzBuzz")
+        elif x % 3 == 0:
+            print("Fizz")
+        elif x % 5 == 0:
+            print("Buzz")
+        else:
+            print(x)
+
+if __name__ == "__main__":
+    fizzbuzz()
```
Comment (charlesrobison) [inline students/laura_denney/lesson02/fizzbuzz.py]: Works well!

File: students/laura_denney/lesson02/grid_printer.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    plus_row1(rc, size)
+
+def askargs():
+    args = ""
+    while args != "exit":
+        print()
+        args = input("How many arguments (1 or 2) would you like for your grid? (type exit to exit): ")
+        if args == "1":
+            num = input("Please choose the size of your grid: ")
+            one_arg_grid(int(num))
```
Comment (charlesrobison) [inline students/laura_denney/lesson02/grid_printer.py]: Code appears to break at this step.

File: students/laura_denney/lesson02/grid_printer.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        elif args == "2":
+            rc = input("Please choose the number of rows/columns: ")
+            size = input("Please choose the number of units: ")
```
Comment (charlesrobison) [inline students/laura_denney/lesson02/grid_printer.py]: Breaks here as well.

File: students/laura_denney/lesson02/grid_printer.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        plus_row1(rc,size)
+        for x in range(size):
+            line_row1(rc, size)
+    plus_row1(rc, size)
+
+def askargs():
+    args = ""
+    while args != "exit":
+        print()
+        args = input("How many arguments (1 or 2) would you like for your grid? (type exit to exit): ")
```
Comment (charlesrobison) [inline students/laura_denney/lesson02/grid_printer.py]: Really like that you added this dialogue!

File: students/laura_denney/lesson02/series.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+#testing block
+#tests that fibonacci() shows correct 8th value in fibonacci sequence
+assert fibonacci(7) == 13, "Error with fibonacci!"
+#tests that lucas() shows correct 8th value in lucas sequence
+assert lucas(7) == 29, "Error with lucas!"
+#tests that sum_series() shows correct 8th value with fibonacci defaults
+assert sum_series(7)== 13, "Error with the generalization to fibonacci!"
+#tests that sum_series() shows correct 8th value with fibonacci defaults
+assert sum_series(7,2,1)== 29, "Error with the generalization to lucas"
```
Comment (charlesrobison) [inline students/laura_denney/lesson02/series.py]: Good assert statements!

**Pull request merged**